src/themes/fonts.js

```diff
@@ -0,0 +1,70 @@
+const font_opensans = new FontFace("OpenSans", `url(https://fonts.googleapis.com/css2?family=Open+Sans:ital,wght@0,300..800;1,300..800&display=swap)`);
```

Comment:
This is nice, I've never actually used a FontFace object before in JS (at least not directly) 👍 We do need to switch this to our own font assets vs the Google Fonts API, though (and camelCase the name instead of snake_case).

Reply:
Do we have a font file for an OpenSans font face?

Reply:
Ah, fair point. Sorry, for some reason I got this mixed up with Calibre. I don't see it in the files from Comms, so I downloaded it from the Google Fonts site (just shared with you in Slack).